esp-radio/MIGRATING-0.15.0.md

```diff
@@ -53,3 +53,12 @@ Provide these symbols:
 ## Scanning Functions
 
 The `scan_with_config_sync_max`, `scan_with_config_sync_max`, `scan_n`, and `scan_n_async` functions have been removed. You can instead use the `scan_with_config_async` or `scan_with_config_sync` funtions while specifying a `max` value in `ScanConfig`.
+
+## Configuration
+
+The `Configuration`, `ClientConfiguration`, `AccessPointConfiguration`, and `EapClientConfiguration` enums have been renamed to `Config`, `ClientConfig`, `AccessPointConfig`, and `EapClientConfig`:
+
+```diff
+- pub enum Configuration { .. }
```

Comment:
we should show the diff for the user's code - not the changes in esp-radio

Reply:
True 👍